services/redis.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sd-redis
+  labels:
+    name: redis
+    app: sd-redis
+spec:
+  ports:
+  - port: 6379
+    name: master
+  clusterIP: None
+  selector:
+    name: redis
+---
+apiVersion: apps/v1beta1
```

Comment:
Is it a preferred practice to use `extensions` instead of `apps`?  Documentation of Statefulset refers to `apps` https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components